sdk/eventhub/azure-eventhub/azure/eventhub/aio/_transport/_pyamqp_transport_async.py

```diff
@@ -55,11 +70,20 @@ async def create_connection_async(**kwargs) -> ConnectionAsync:
         :return: The created ConnectionAsync.
         :rtype: ~pyamqp.aio.Connection
         """
-        endpoint = kwargs.pop("endpoint")
         host = kwargs.pop("host")  # pylint:disable=unused-variable
         auth = kwargs.pop("auth")  # pylint:disable=unused-variable
-        network_trace = kwargs.pop("debug")
-        return ConnectionAsync(endpoint, network_trace=network_trace, **kwargs)
+        network_trace = debug
+        return ConnectionAsync(
+            endpoint,
+            container_id=container_id,
+            max_frame_size=max_frame_size,
+            channel_max=channel_max,
+            idle_timeout=idle_timeout,
+            properties=properties,
+            idle_timeout_empty_frame_send_ratio=remote_idle_timeout_empty_frame_send_ratio,
+            network_trace=network_trace,
```

Comment:
error_policy + encoding?

Reply:
same as the sync one. Not used by pyamqp, could pass it ?